Gemfile.lock

```diff
@@ -65,9 +65,11 @@ GEM
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
+    webrick (1.8.1)
 
 PLATFORMS
   ruby
+  x64-mingw-ucrt
```

Comment:
Can you explain what webrick and this new platform would help with? Or were those just changes for your local development environment?

Reply:
webrick and x64-mingw-ucrt were to make jekyll execute on my machine (yep, Windows). I can pull that change out of the PR.